### ISSUE TITLE
Dynamic Trees Support

### ISSUE THREAD

**Issue by ghost** [octarine-noise/BetterFoliage#165]
no rounded logs for Dynamic Trees mod's trees...
![2018-01-09_21 12 28](https://user-images.githubusercontent.com/24831406/34735829-db64ec22-f581-11e7-97c2-a45356eec062.png)
could it be the problem at their side?

**Comment by ghost** [comment]
Hey there.  I'm the author of dynamic trees.  The "problem" is on my side.  I do a custom model that I generate internally(no json) because the model would be unmanageable otherwise.  So it's pretty much on me to implement this and I have no plans for ever supporting rounded tree trunks.  The feature would be more complicated than I'm willing to put up with.  Sorry.

**Comment by octarine-noise** [comment]
It might be possible to do special support from my side if the blockstate is not too complicated, but it's likely to be a lot of work.
I'm not sure about this yet, I may just pass on it, but even if I do it it'll be only for the maximum-size trunks. I found that thinner columns don't look good when rounded anyway.

**Comment by ghost** [comment]
yeah agreed, i recommend only the main logs that on the middle.(biggest ones) (be rounded)
